components/molecules/social-media.vue

```diff
@@ -3,19 +3,37 @@
     <span>Volg Stad Gent op sociale media</span>
     <div class="social-list">
       <ul class="">
-        <li><a href="https://www.facebook.com/stadsbestuurgent"><i aria-hidden="true"
-                                                                   class="icon-facebook"/><span
-                                                                     class="visually-hidden">Facebook</span></a></li>
-        <li><a href="https://www.twitter.com/stadgent"><i aria-hidden="true" class="icon-twitter"/><span
-          class="visually-hidden">Twitter</span></a></li>
-        <li><a href="https://www.linkedin.com/company/stad-gent"><i aria-hidden="true"
-                                                                    class="icon-linkedin"/><span
-                                                                      class="visually-hidden">LinkedIn</span></a></li>
-        <li><a href="https://www.youtube.com/channel/UCmJLf8SFG0D0GTd6LSSTQLw"><i aria-hidden="true"
-                                                                                  class="icon-youtube"/><span
-                                                                                    class="visually-hidden">Youtube</span></a></li>
-        <li><a href="https://www.instagram.com/stadgent/"><i aria-hidden="true" class="icon-instagram"/><span
-          class="visually-hidden">Instagram</span></a></li>
+        <li>
+          <a href="https://www.facebook.com/stadsbestuurgent"><i aria-hidden="true"
```

Comment:
Formatting should be improved for this file. This isn't exactly readable 😅

```html
<li>
	<a href="https://www.facebook.com/stadsbestuurgent">
		<i aria-hidden="true" class="icon-facebook"/>
		<span class="visually-hidden">Facebook</span>
	</a>
</li>
```

Reply:
Yes... the linting rules have changed.
Guess lintfix wasn't the prettiest solution.